cellrank/tools/estimators/_gppca.py

```diff
@@ -933,15 +938,19 @@ def set_categories(group):
         meta_assignment["assignment"] = (
             meta_assignment["assignment"].astype(str).astype("category")
         )
+        meta_assignment["assignment"].cat.set_categories(["0", "1", "2"], inplace=True)
```

Comment:
Why do you hardcode this here?

Reply:
This was when I was testing this and I forgot it there, sorry.